ecommerce/shopper_experience/payment_acquirer.rst

```diff
@@ -1,78 +1,70 @@
-  Authorize.net).
+And, if the user clicks on the link to the journal entry, they will be taken to that journal
+entry's page, showcasing all the pertinent details surrounding this specific transaction
 
-* *Done*: the payment is authorized and captured. The order has been confirmed.
-
-* *Error*: an error has occured during the transaction. 
-  The customer needs to retry the payment.
-  The order is still in draft.
-
-* *Cancelled*: when the customer cancels the payment in the payment acquirer form.
-  They are taken back to Odoo in order to modify the order.
+.. image:: media/reconciled-journal-entry.png
+   :align: center
+   :alt: payment is confirmed in the chatter of sales order
 
 .. note:: Specific messages are provided to your customers for every
-   payment status, when they are redirected to Odoo after the transaction.
-   To edit such messages, go to the *Messages* tab of the payment
+   payment status whenever they are redirected to Odoo after the transaction.
+   To edit these messages, go to the *Messages* tab of the payment
    method.
 
+Automatically generate invoices at order
+========================================
 
-Auto-validate invoices at order
-===============================
-
-When the order is confirmed you can also have an invoice automatically issued
-and paid. This fully-automated made for businesses that invoice 
+When the order is confirmed, you can also have an invoice automatically issued
+and paid. This fully-automated feature is designed for businesses that invoice 
 orders straight on.
 
-.. image:: media/payment_invoice.png
-    :align: center
+To do this, go to the *Invoicing* section of the *Settings* in the Website application,
+and select *Invoice what is ordered* and then select *Automatic Invoices*, which will
+generate an invoice automatically when the online payment is confirmed.
+
+.. image:: media/automatic-invoice.png
+   :align: center
+   :alt: example of automatic invoice
 
 .. note::
-   If you choose this mode you are requested to select a payment journal in order to record payments
-   in your books. This payment is automatically reconcilied with the invoice, marking it as paid.
-   Select your **bank account** if you get paid immediately on your bank account. If you don't you
-   can create a specific journal for the payment acquirer (type = Bank). That way, you can track
+   If you choose this mode, you are requested to select a payment journal, in order to record 
+   payments in your books. This payment is automatically reconciled with the invoice, marking it as
+   paid. Select your **bank account** if you get paid immediately on your bank account. If you don't 
+   you can create a specific journal for the payment acquirer (type = Bank). That way, you can track
    online payments in an intermediary account of your books until you get paid into your bank
    account (see :doc:`../../general/payment_acquirers/payment_acquirers`).
```

Comment:
This note is not about about generating invoices automatically.
I believe it is linked to the next point "capture payment". Also, I think that the part about creating new journals for payment acquirers isn't completely correct anymore, since you would configure which journal to use when you first activate a new payment acquirer. (also, the new journal is now created automatically by odoo, when you do that).

I would simply remove this note.
```suggestion
```